ClientApp/src/app/courses/word-table/word-table.component.html

```diff
@@ -0,0 +1,77 @@
+
+<div class="mat-elevation-z8 table-control-container"  *ngIf="course.words.length > 0 && displayHeader">
+    <select class="table-control-select" (change)="changeSelectedTableAction($event)">
+        <option value="null">(Select Action)</option>
+        <option value="suspend">Suspend Word(s)</option>
+        <option value="restore">Restore Word(s)</option>
+        <option value="edit">Edit Word</option>
+        <option value="reset">Reset Word(s)</option>
+    </select>
+    <button type="button" class="table-control-submit" (click)="executeTableAction()">Submit</button>
+</div>
+<div class="mat-elevation-z8 word-table" *ngIf="course.words.length > 0;">
+    <table mat-table [dataSource]="tableData">
+
+        <!-- Japanese Vocab Column -->
+        <ng-container matColumnDef="Japanese">
+            <th mat-header-cell *matHeaderCellDef> Japanese Vocab </th>
+            <td mat-cell *matCellDef="let word">
+                <input *ngIf="displayHeader" class="word-checkbox" type="checkbox" (change)="toggleWordStatus(word)">
+                {{word.japaneseVocab}}
+                [{{word.kanaVocab}}]
+            </td>
+        </ng-container>
+    
+        <!-- English Vocab Column -->
+        <ng-container matColumnDef="English">
+            <th mat-header-cell *matHeaderCellDef> English </th>
+            <td mat-cell *matCellDef="let word">
+                <ng-container *ngIf="displayAnswers; else hideWordAnswers">
+                    <span class="answer visible" id={{word.id}} #wordAnswer *ngIf="displayAnswers">{{word.englishVocab}}</span>
+                    <mat-icon class="visibility-icon" #visibilityIcon (click)="toggleVisibility(word, 'word', visibilityIcon)">visibility_on</mat-icon>
+                </ng-container> 
+
+                <ng-template #hideWordAnswers>
+                    <span class="answer hidden" id={{word.id}} #wordAnswer>{{word.englishVocab}}</span>
+                    <mat-icon class="visibility-icon" #visibilityIcon (click)="toggleVisibility(word, 'word', visibilityIcon)">visibility_off</mat-icon>
+                </ng-template>
+            </td>
+
+
+        </ng-container>
+    
+        <!-- Sentence Column -->
+        <ng-container matColumnDef="Sentence">
+            <th mat-header-cell *matHeaderCellDef> Sentence </th>
+            <td mat-cell *matCellDef="let word">
+                <ng-container *ngIf="displayAnswers; else hideSentenceAnswers">
+                    <p *ngFor="let sentence of word.sentences" class="answer visible" id={{word.id}} #sentenceAnswer>
+                        <span
+                        *ngIf="sentence.sentenceText != null &&
+                        sentence.sentenceType.type != 'Furigana'"
+                        >
```

Comment:
```suggestion
                        <span
                            *ngIf="sentence.sentenceText != null
                            && sentence.sentenceType.type != 'Furigana'"
                        >
```

Reply:
Not a huge fan of string comparisions in templates. Can you make all of these comparisons comparisons to constant strings? especially if you use them elsewhere, then that should live in a constants file somewhere like `sentance-type-constant.ts` f.ex